justfile

```diff
@@ -0,0 +1,2 @@
+build_plow:
+    cargo build --bin plow && cp ./target/debug/plow ~/bin && echo "Installed plow to ~/bin"
```

Comment:
Maybe make it a release build instead 🙃

Reply:
Definitely! :)

Reply:
https://github.com/field33/plow/compare/c9323fbb50b3f0fb7394b90c009a5d78064018d7..9346e090ad154c954b5f23651a3925e4deedb1e9